CheckPoint FW v2 - moved methods into client (#14115)
demisto/content: Packs/CheckpointFirewall/ReleaseNotes/2_1_0.md

#### Integrations
##### CheckPoint Firewall v2
- Moved non-`Client` methods into the client, for better proxy support. 

> - Better error handling

Reviewer: @dorschw - These notes are more dev-oriented. The notes should be more aligned with what the customer is experiencing. Try to think how they might experience the change made - you can look at some best practices here: https://xsoar.pan.dev/docs/documentation/release-notes#examples-and-best-practices

Reviewer: Also stay aligned with the formats in the docs please

Author: done, can you take another look please?

> ##### CheckPoint Firewall v2
> - Fixed issues with logging in and calling commands via proxy connection.
> - Improved error handling.

Reviewer 2: ```suggestion
##### CheckPoint Firewall v2
- Fixed issues when logging in and using the integration with **proxy** connection.
```
